backend/src/schema/resolvers/searches.spec.js

```diff
@@ -0,0 +1,452 @@
+                await expect(query({ query: searchQuery, variables })).resolves.toMatchObject({
+                  data: {
+                    findResources: expect.arrayContaining([
+                      {
+                        __typename: 'Post',
+                        id: 'a-post',
+                        title: 'Beitrag',
+                        content: 'Ein erster Beitrag',
+                      },
+                      {
+                        __typename: 'Post',
+                        id: 'b-post',
+                        title: 'Aufruf',
+                        content: 'Jeder sollte seinen Beitrag leisten.',
+                      },
+                    ]),
+                  },
+                })
+              })
+            })
+          })
+
+          describe('a post that contains a hyphen between two words and German quotation marks', () => {
+            describe('hyphens in query', () => {
+              it('will be treated as ordinary characters', async () => {
+                variables = { query: 'tee-ei' }
+                await expect(query({ query: searchQuery, variables })).resolves.toMatchObject({
+                  data: {
+                    findResources: [
+                      {
+                        __typename: 'Post',
+                        id: 'g-post',
+                        title: 'Zusammengesetzte Wörter',
+                        content: `Ein Bindestrich kann zwischen zwei Substantiven auch dann gesetzt werden, wenn drei gleichlautende Buchstaben aufeinandertreffen. Das ist etwa bei einem „Teeei“ der Fall, das so korrekt geschrieben ist. Möglich ist hier auch die Schreibweise mit Bindestrich: Tee-Ei.`,
+                      },
+                    ],
+                  },
+                })
+              })
+            })
+
+            describe('German quotation marks in query', () => {
+              it('will be treated as ordinary characters', async () => {
+                variables = { query: '„teeei“' }
+                await expect(query({ query: searchQuery, variables })).resolves.toMatchObject({
+                  data: {
+                    findResources: [
+                      {
+                        __typename: 'Post',
+                        id: 'g-post',
+                        title: 'Zusammengesetzte Wörter',
+                        content: `Ein Bindestrich kann zwischen zwei Substantiven auch dann gesetzt werden, wenn drei gleichlautende Buchstaben aufeinandertreffen. Das ist etwa bei einem „Teeei“ der Fall, das so korrekt geschrieben ist. Möglich ist hier auch die Schreibweise mit Bindestrich: Tee-Ei.`,
+                      },
+                    ],
+                  },
+                })
+              })
+            })
+          })
+
+          describe('a post that contains a simple mathematical exprssion and linebreaks', () => {
+            describe('query a part of the mathematical expression', () => {
```

Comment:
this is cool, and seeing the docs, I can see how this is possible. I wonder if this is an actual use case, but I guess it doesn't hurt to have a test case for it?